webpack.prod.js

```diff
@@ -20,7 +20,8 @@ module.exports = merge(commons, {
           path.resolve(__dirname, "src"),
           path.resolve(__dirname, "node_modules/esn-frontend-common-libs"),
           path.resolve(__dirname, "node_modules/esn-frontend-mailto-handler"),
-          path.resolve(__dirname, "node_modules/esn-frontend-account-inbox")
+          path.resolve(__dirname, "node_modules/esn-frontend-account-inbox"),
+          path.resolve(__dirname, "node_modules/esn-frontend-inbox")
```

Comment:
What is really missing to have to do this? Can you explain?

Reply:
Runing en mode production throws this error :
eProvider <- e <- inboxRestangular <- inboxUsersIdentitiesClient <- identitiesService
so i added esn-frontend-inbox because it is required by account-inbox